hubploy/auth.py

```diff
@@ -222,26 +222,24 @@ def cluster_auth_aws(deployment, project, cluster, zone, service_key):
 
     This changes *global machine state* on what current kubernetes cluster is!
     """
+    print(deployment, project, cluster, zone, service_key)
```

Comment:
Let's take this out?

Reply:
sorry - i can take out the entire auth.py changes for this PR, I didn't think i'd committed them....